libscidavis/src/AssociationsDialog.cpp

```diff
@@ -131,7 +131,11 @@ void AssociationsDialog::changePlotAssociation(int curve, const QString& text)
   if (c->plotAssociation() == text)
     return;
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+  QStringList lst = text.split(",", Qt::SkipEmptyParts);
+#else
   QStringList lst = text.split(",", QString::SkipEmptyParts);
+#endif
```

Comment:
Wouldn’t it be better (less intrusive) to use macros?
For instance:
```suggestion
#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
#define QT_SKIPEMPTYPARTS Qt::SkipEmptyParts
#define QT_KEEPEMPTYPARTS Qt::KeepEmptyParts
#else
#define QT_SKIPEMPTYPARTS QString::SkipEmptyParts
#define QT_KEEPEMPTYPARTS QString::KeepEmptyParts
#endif
```

Reply:
> Wouldn’t it be better (less intrusive) to use macros?
> For instance:
> ```suggestion
> #if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
> #define QT_SKIPEMPTYPARTS Qt::SkipEmptyParts
> #define QT_KEEPEMPTYPARTS Qt::KeepEmptyParts
> #else
> #define QT_SKIPEMPTYPARTS QString::SkipEmptyParts
> #define QT_KEEPEMPTYPARTS QString::KeepEmptyParts
> #endif
> ```

And where to put these macros?

Reply:
Either in a common header or in the '*.cpp' files that use the macros. 
That is what I did for the python2/python3 switch in files PythonScript.cpp, PythonScripting.cpp, scidavis.sip